earn/src/index.tsx

```diff
@@ -11,8 +10,9 @@ import App from './App';
 if (process.env.REACT_APP_SENTRY_DSN) {
   Sentry.init({
     dsn: process.env.REACT_APP_SENTRY_DSN,
-    integrations: [new BrowserTracing()],
-
+    integrations: [new Sentry.BrowserTracing()],
+    // TODO: Automate this
```

Comment:
I'm okay with merging this with the TODO, but I would like to hear what the plan is to address it in the future. Like in broad strokes how would that work?

Reply:
Sentry has a GitHub Action we can use to automate this.